controller/destination/listener.go

```diff
@@ -99,6 +99,9 @@ func (l *endpointListener) toWeightedAddr(address common.TcpAddress) *pb.Weighte
 				podFound = true
 				metricLabelsForPod = pkgK8s.GetOwnerLabels(pod.ObjectMeta)
 				metricLabelsForPod["pod"] = pod.Name
+				if pod.Labels[pkgK8s.ControllerNSLabel] != "" {
+					metricLabelsForPod["conduit_io_control_plane_ns"] = pod.Labels[pkgK8s.ControllerNSLabel]
```

Comment:
I have a slight preference for modifying the `GetOwnerLabels` func in pkg/k8s/labels.go to set this automatically. I think if you update the `proxyLabels` slice in that file to include `ControllerNSLabel`, and the update `toOwnerLabel` to work with that label, it should all Just Work. If you do that, can you also update pkg/k8s/labels_test.go to test the new behavior?

And I vote to call this label "conduit_ns", unless there's some aspect #386 that requires the label be "conduit_io_control_plane_ns".

Reply:
i do think that the dst metrics should correspond to pod labels. so if we're giong to use `dst_conduit_ns` here, we should use `conduit_ns` elsewhere

Reply:
I updated `GetOwnerLabels` to set this label as well.

I refrained from changing `conduit_io_control_plane_ns` becasue I think the proxy sets those labels from whatever we label the injected pods (i.e. doesn't go through GetOwnerLabels), so they won't match if I only change the name here. (Though if someone can point me to where those are being set I could change them too - I couldn't find it).

Reply:
Ah, ok -- in that case I definitely agree that the label should be `conduit_io_control_plane_ns`.

---

For my own understanding, I looked up all of the labels on the `repsonse_total` metric from the web deployment in the emojivoto app. Here they are:

```
response_total{
  app="web-svc",
  authority="voting-svc.emojivoto:8080",
  classification="failure",
  conduit_io_control_plane_ns="conduit",
  deployment="web",
  direction="outbound",
  dst_deployment="voting",
  dst_namespace="emojivoto",
  dst_pod="voting-65fdc8dd85-l4k4l",
  dst_pod_template_hash="2198748841",
  dst_service="voting-svc",
  grpc_status_code="2",
  instance="172.17.0.11:4191",
  job="conduit-proxy",
  namespace="emojivoto",
  pod="web-5548b7dbc7-sxjwn",
  pod_template_hash="1104638673",
  status_code="200"
}
```

I have to say that `conduit_io_control_plane_ns` feels like a bit of an outlier. Most of the rest of the labels are concise and don't include the word "conduit", and the "namespace" and "dst_namespace" labels don't use the short form "ns". Overall, I think my preference would be to rename the controller namespace label to "controller_namespace", and change this label to be "dst_controller_namespace", but that's way outside the scope of this change, and I agree that they should match, regardless of whatever the string is.